src/fmu/tools/fipmapper/fipmapper.py

```diff
@@ -167,6 +171,21 @@ def _fipdata_from_webvizyaml(self, yamldict: dict):
     def _fips2regions(self, fips: List[int]) -> List[List[str]]:
         return [self.fip2region(fip_int) for fip_int in fips]
 
+    def get_regions(self) -> List[str]:
```

Comment:
These should all get their own tests with both happy and not-so-happy path being tested...

Reply:
Fixed.